lib/src/clojure_lsp/feature/file_management.clj

```diff
@@ -83,29 +83,33 @@
         usage-signature (juxt :to :name)]
     (find-changed-elems-by usage-signature old-var-usages new-var-usages)))
 
+(defn reference-filenames [filename old-local-analysis new-local-analysis db]
+  (let [project-analysis (into {} (q/filter-project-analysis-xf db) (:analysis @db))
+        source-paths (settings/get db [:source-paths])
+        changed-var-definitions (find-changed-var-definitions old-local-analysis new-local-analysis)
+        references-filenames (->> changed-var-definitions
+                                  ;; TODO: switch from nested-loop join to hash join
+                                  (mapcat #(q/find-references project-analysis % false db))
+                                  (map :filename))
+        changed-var-usages (find-changed-var-usages old-local-analysis new-local-analysis)
+        definitions-filenames (->> changed-var-usages
+                                   ;; TODO: remove usages of external namespaces (clojure.core, etc.) here
+                                   ;; TODO: switch from nested-loop join to hash join
+                                   (keep #(q/find-definition project-analysis % db))
+                                   (filter (fn [d]
+                                             ;; TODO: this excludes "private calls", but they are counted in code lens, so maybe shouldn't exclude
```

Comment:
Yeah, I thought code elens use to work without this for some reason I can't recall, I think if we try removing it and confirm code lens are updated as well, we can safely remove this private check

Reply:
@ericdallo do you mind if I remove the private check as part of #892? If I remove it from this PR it'll cause conflicts with that one.

Reply:
np